### PR TITLE
Get sort query parameter to work again

### PR DESCRIPTION
**Pull request by evz**
As mentioned in #51, sorting the API responses doesn't work. When one digs into why it was failing, it had to do with the way that Django was attempting to construct the `DISTINCT ON` clause for PostgreSQL. Since PostgreSQL requires that the `ORDER BY` portion of the query have fields in the same order as the fields in the `DISTINCT ON` portion of the query, and Imago just appeared to be constructing the query one step at a time, Django didn't really know what to do in that situation.

This PR just removes the `DISTINCT ON` thing since there really shouldn't be a situation where there is more than one object with the same ID. Does this seem sane?

### CODE REVIEW

File: imago/management/commands/loadmappings.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
 
 def load_mapping(boundary_set_id, key, prefix, boundary_key='external_id', ignore=None, quiet=False):
+
```
Comment (jpmckinney) [inline imago/management/commands/loadmappings.py]: Why new line?

File: imago/management/commands/loadmappings.py [hunk 2]
```diff
@@ -19,9 +20,11 @@ def load_mapping(boundary_set_id, key, prefix, boundary_key='external_id', ignor
     for div in Division.get('ocd-division/country:' + settings.IMAGO_COUNTRY).children(levels=100):
         if div.attrs[key]:
             geoid_mapping[div.attrs[key]] = div.id
-
+        else:
+            geoid_mapping[div.id] = div.id
+    
     print('processing', boundary_set_id)
-
+
```
Comment (jpmckinney) [inline imago/management/commands/loadmappings.py]: Revert whitespace addition.

**Comment by jpmckinney** [comment]
Can you split out the following into separate PRs:

* auth helper fix
* schema fix
* exposing membership contact details

**Comment by jpmckinney** [comment]
Hmm, nevermind - (some of) those commits already exist in master, but for some reason show up in the PR.

**Comment by jpmckinney** [comment]
@paultag added that line in https://github.com/opencivicdata/imago/commit/d977bba960c9596da2dafecf596f6d3338b08d92 so I assume there's a reason for it.

**Comment by evz** [comment]
@jpmckinney Yeah, I was kinda scratching my head about that, too. As far as I could tell, I was keeping my fork synced with opencivicdata.

**Comment by evz** [comment]
@jpmckinney I figure it's there for a reason. It's just causing Django to not know what to do when you're wanting to sort. I was hoping we could get to the bottom of it here.

**Comment by jpmckinney** [comment]
I mentioned paultag, since he would know what's going on - but I figure the preceding lines must sometimes return duplicates.

**Comment by fgregg** [comment]
ping @paultag

**Comment by fgregg** [comment]
@jpmckinney, here's the word from paultag

```
11:45 <+paultag> I'm afraid I can't gain context at the moment, but it's been 
                 on my queue
11:45 <+paultag> if my actions looked irrational, ignore them and revert
```

File: imago/serialize.py [hunk 3]
```diff
@@ -85,12 +85,13 @@ def sfilter(obj, blacklist):
...
 
 
-LINK_SERALIZE = dict([
+LINK_SERIALIZE = dict([
     ("note", {}),
     ("url", {}),
 ])
 
 
+
```
Comment (jpmckinney) [inline imago/serialize.py]: No need for newline

File: imago/management/commands/loadmappings.py (hunk 2 above)
Comment (jpmckinney) [inline imago/management/commands/loadmappings.py]: Extra whitespace. Remove the new newlines as well.

**Comment by jpmckinney** [comment]
Actually, can you rebase so that all the old commits don't show up here? I don't know what I'm commenting on.

**Comment by fgregg** [comment]
Replaced by #77

**Pull request closed without merging**